src/core/runner.ts

```diff
@@ -466,6 +467,9 @@ export default class Runner implements RunnerInfo {
       (journey.cb ?? journey.callback)({ params: options.params } as any);
       const monitor = journey.monitor ?? journey?._getMonitor();
       monitor.update(this.#monitor?.config);
+      monitor.setSpaces(
```

Comment:
This should not be required as the .update call should handle the changes from the config. Please check once.